src/main.rs

```diff
@@ -17,15 +20,21 @@ fn min_max(a: i16, b: i16) -> (i16, i16) {
     }
 }
 
-fn build_guides(screen: xcb::Rectangle, pt: xcb::Point, width: u16) -> [xcb::Rectangle; 2] {
+fn build_guides(screen: xproto::Rectangle, pt: xproto::Point, width: u16) -> [xproto::Rectangle; 2] {
     [
-        xcb::Rectangle::new(
-            pt.x() - width as i16 / 2,
-            screen.x(),
-            width,
-            screen.height(),
-        ),
-        xcb::Rectangle::new(screen.y(), pt.y() - width as i16 / 2, screen.width(), width),
+        xproto::Rectangle {
+            x: pt.x - width as i16 / 2,
```

Comment:
```suggestion
            x: pt.x - (width as i16) / 2,
```

I dont like the way this looks without parenthesis

Reply:
I like it.

Reply:
i think it's fine without parens